nodes/src/nodes/llm_base/IInstance.py

```
@@ -25,13 +25,34 @@
+            elif provider.startswith('llm_'):
+                provider = provider[4:]
+            self.__class__._resilience_policy = create_resilience_policy(provider)
+        return self._resilience_policy
+
+    def _question(self, question: Question) -> Answer:
+        # Execute the LLM call through the resilience policy (circuit breaker + retry).
+        policy = self._get_resilience_policy()
+        try:
+            answer = policy.execute(self.IGlobal._chat.chat, question)
+        except CircuitBreakerOpenError:
+            raise
```

Ruff (0.15.7)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)